src/Initializer/InitProcedure/InitMesh.cpp

```diff
@@ -175,5 +186,14 @@ void seissol::initializer::initprocedure::initMesh() {
   // Call the post mesh initialization hook
   seissol::Modules::callHook<seissol::POST_MESH>();
 
-  logInfo(seissol::MPI::mpi.rank()) << "End init mesh.";
+  logInfo(commRank) << "End init mesh.";
+
+  if ((seissolParams.mesh.showEdgeCutStatistics) && (commSize > 0)) {
```

Comment:
Shouldn't this be > 1 or am I missing something?

Reply:
You're right

Reply:
done